### DISCUSSION TITLE
Somos 3k users!

### DISCUSSION THREAD

**Discussion by AlencarGabriel** [AlencarGabriel/ProtheusDoc-VsCode#48] [General]
# Somos 3k users :star_struck: :raised_hands:

É com grande prazer e alegria que anuncio que chegamos a marca de 3k usuários que instalaram a extensão ProtheusDoc for VsCode!

Para alguns talvez seja um número irrelevante, comparado a outras extensões existentes.

Mas para mim é uma satisfação poder ter contribuído em mais este projeto para benefício de toda comunidade TOTVS Protheus.

### Um pouquinho da história deste projeto:

No início deste ano (2020) eu estava usando o TDS (Eclipse) apenas para gerar as documentações ProtheusDoc, e me perguntava quando que haveria um recurso parecido para o VsCode, visto que todos já estavam migrando para este editor.

Logo depois pensei, porque não usar meus conhecimentos já adquiridos criando a extensão [advpl-switch-project](https://github.com/AlencarGabriel/advpl-switch-project) e contribuindo com a [advpl-vscode](https://github.com/totvs/advpl-vscode), e criar mais uma extensão, agora para prover o ProtheusDoc, utilizando o máximo de recursos que as API's do VsCode proveem.

Foi ai então que nasceu a ideia de criar a extensão, e com ela pude gastar algumas horas do meu tempo livre para ajudar um pouquinho mais a comunidade.

No início pensei que quase ninguém usaria, mesmo assim fiz um [artigo em meu blog](https://gabrielalencar.dev/2020/01/29/ProtheusDoc-for-VsCode/) apresentando a extensão e o ProtheusDoc para aqueles que não conheciam. Pedi a alguns amigos que compartilhassem, e de lá pra cá só vi os números de acesso e downloads subirem. 

Ao mesmo tempo que o número de downloads subia, comecei a receber feedbacks, avaliações, sugestões, reclamações, reports e até mesmo Pull Requests! Cada uma dessas contribuições ajudou a fortalecer este projeto, e ainda a me sentir útil nesta imensa comunidade!

---

Agradeço a você, que de alguma forma vem contribuindo com este projeto, mesmo que seja apenas usufruindo! 

Por isso te peço que [compartilhe e avalie este projeto](https://marketplace.visualstudio.com/items?itemName=AlencarGabriel.protheusdoc-vscode), colabore mandando sugestões ou reportando algum problema. Assim ajudaremos mais pessoas e faremos essa família crescer cada vez mais. 

E conte comigo para continuar contribuindo como for possível! :handshake:

|**ProtheusDoc for VsCode** |
|:--:|
| ![Installs](https://vsmarketplacebadge.apphb.com/installs/AlencarGabriel.protheusdoc-vscode.svg) ![Downloads](https://vsmarketplacebadge.apphb.com/downloads/AlencarGabriel.protheusdoc-vscode.svg) ![Rating](https://vsmarketplacebadge.apphb.com/rating-star/AlencarGabriel.protheusdoc-vscode.svg) |
| ![ProtheusDoc VsCode](https://user-images.githubusercontent.com/10109480/102130976-4a401c80-3e30-11eb-9e45-8f0402b344ef.png) |

**Comment by edunxp** [comment]
Apesar do post ser antigo, vai meus parabéns.

**Reply by AlencarGabriel** [reply]
Obrigado! 🎉 E mais que dobramos esse número já rs

**Comment by CBAA-Aguiar** [comment]
Bom dia.
Estou usando, acredito, uma nova versão v0.10.0 A qual taz as seguntes informações: {Protheus.doc}
  
 @type  
 @author 
 @since 
 @version
 @return 
 @history
Mas isso esta me dando um problema pq qdo uso o @history e na frente coloco as atualizações ele traz uma linha chamada OUTRAS INFORMAÇÕES com o mesmo numero de linhas que coloco no @history. Exemplo se coloco 2 @history, cadas um repsentando uma alteração no programa dus linhas com o conteudo history | [object Object] na linha abaixo de OUTRAS INFORMAÇÕES. Poderia me explicar  porque?
-- | --

**Reply by AlencarGabriel** [reply]
Transferido para issue: #85
